### ISSUE TITLE
Changing the title text on a block with a "multiple inputs" slot clears args in call sites

### ISSUE THREAD

**Issue by dlnelson** [jmoenig/Snap--Build-Your-Own-Blocks#1962]
I hesitate even to post this because of Jens's diatribe in #1054 (worse, I sympathize deeply with the sentiment).  So, at the risk of sounding like an ungrateful user . . .

Repro steps:
-   Create a custom block with a "multiple inputs" argument slot.  For example "test args...".
-   Call the block in a few places, with differing numbers of arguments (e.g., "test", "test foo", "test foo bar").
-   Edit the block definition and change only the title text (for example to "test2 args...").

Expected:
  Call sites reflect the title text change, with the passed arguments unaffected (e.g. "test2", "test2 foo", "test2 foo bar").

Actual:
  All call sites are reset to "test2 _".

I think this is somewhat worse that #1054, because I was only changing the title text.  As someone who has tripped over #1054 and understands that limitation (and the reasons behind it), I wasn't prepared for this one.

Fortunately, I'm superstitious and paranoid, so I had a backup that I was able to recover from.

(Rest assured that I'm well aware that Snap is awesome.)

**Comment by brianharvey** [comment]
Nah, don't worry, Jens doesn't yell at users that way, just at me.  If he gets annoyed about you filing this, he'll just yell at me in email. ☺ (Anyway, I'm right about #1054, and eventually I'll convince him.)
